### ISSUE TITLE
possible typo in movq (and maybe movd is as well)

### ISSUE THREAD

**Issue by robertmuth** [asmjit/asmdb#22]
movq is the only instruction using this descriptor:   r64[63:0]/m64
"movq"             , "W:xmm[63:0], r64[63:0]/m64" 

should this be just r64/m64 ? Especially since the MR variant looks like:
"movq"             , "W:r64/m64, xmm[63:0]"  

movd seems also suspicious:
"movd"             , "W:r32[31:0]/m32, xmm[31:0]"       
"movd"             , "W:xmm[31:0], R:r32[31:0]/m32"

but the use of r32[31:0] sees to be more widespread,

**Comment by kobalicek** [comment]
Yeah, I think you are right and that the indexes are redundant in these cases, I mean r32[31:0] is essentially r32

**Comment by kobalicek** [comment]
BTW I 'm planning to fix some of the issues that you have reported, I just didn't find a time window for that yet
